openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -111,19 +111,22 @@ def extract_dates_from_course(course):
             # unless that item already has a relative date set
             for _, section, weeks_to_complete in spaced_out_sections(course):
                 section_date_items = []
+                section_due_date = timedelta(weeks=1)
```

Comment:
Maybe add a comment here that the section_due_date will end up being the max of all due dates of its subsections?